css/glpi-gantt.scss

```diff
@@ -0,0 +1,225 @@
+/**
```

Comment:
It would be preferable to encapsulate all these styles into a `.gantt-wrapper` class, or something like that, to ensure styles will be only applied to the gantt view.